interfaces/matlab_experimental/readme.md

```diff
@@ -7,9 +7,10 @@ function calling from Cantera CLib.
 
 1. Install Matlab (any release newer than R2008a).
 2. Compile Cantera from Source and install in your Conda environment, as directed in
-   this link. https://cantera.org/install/compiling-install.html. The experimental
-   Matlab Toolbox does not require a SCons option to install at this moment since it's
-   stand-alone. It also does not require the current Matlab Toolbox to be installed.
+   this link. https://cantera.org/stable/develop/compiling-install.html. The
+   experimental Matlab Toolbox does not require a SCons option to install at this moment
+   since it's stand-alone. It also does not require the current Matlab Toolbox to be
```

Comment:
This is no longer accurate (the legacy matlab toolbox was removed).

Reply:
Resolved separately in #1817.